build.sbt

```diff
@@ -80,3 +94,21 @@ fork in Test := true
 
 // forward stdin/out to fork, so the OpenIE CLI can be run in sbt.
 connectInput in run := true
+
+// The OpenIE codebase has many insignificant style warnings that appear
+// whenever the code is compiled. There are also some important-sounding
+// warnings which don't seem to affect the behavior but should be
+// fixed eventually:
+//
+//   Classes must override both equals and hashCode, not just one
+//   Line is more than 100 characters long
+//   Tab in Scala file
+//   Unusual class name; consider renaming
+//   illegal.imports.message
```

Comment:
I wondered that too. The `enableLineLimit` var is used excplictly by `configWithLineCheck`. I don't see code for other style checks, so it doesn't look like I can pick out the style checks a la cart -- at least not without updating StylePlugin.scala.

Yeah, we can fix all the style problems now but I wanted to avoid getting into the weeds at this moment. Here's what the warnings are about:

- the tab appears in a regex string (maybe "\t" is a good replacement)
- updating the hashCode/equals problems can change behavior which I wanted to isolate.
- changing unusual class names are possibly backwards incompatible changes.
- what's illegal about importing scala.collection.JavaConversions._?

I'll press on if you guys would like.